parts/linux/cloud-init/artifacts/components.json

```diff
@@ -432,88 +49,28 @@
             "versions": [
               "1.6.26-5.cm2"
             ]
-          }
-        },
-        "default": {
-          "current": {
+          },
+          "v2": {
             "versions": [
-              "1.7.20"
+              "1.6.26-5.cm2"
             ]
-          }
-        }
-      }
-    },
-    {
-      "name": "cni-plugins",
-      "downloadLocation": "/opt/cni/downloads",
-      "downloadURIs": {
-        "default": {
-          "current": {
-            "versions": [
-              "1.4.1"
-            ],
-            "downloadURL": "https://acs-mirror.azureedge.net/cni-plugins/v${version}/binaries/cni-plugins-linux-${CPU_ARCH}-v${version}.tgz"
-          }
-        }
-      }
-    },
-    {
-      "name": "azure-cni",
-      "downloadLocation": "/opt/cni/downloads",
-      "downloadURIs": {
-        "default": {
-          "current": {
+          },
+          "v3": {
             "versions": [
-              "1.4.54",
-              "1.5.28",
-              "1.5.32"
-            ],
-            "downloadURL": "https://acs-mirror.azureedge.net/azure-cni/v${version}/binaries/azure-vnet-cni-linux-${CPU_ARCH}-v${version}.tgz"
+              "1.7.13"
+            ]
+          },
+          "v3-kata": {
+            "versions": []
```

Comment:
Kata team raised a request. They don't want `containerd` to be (re)installed because it was already installed somewhere else. I am thinking adding a "kata" at the os release version level. 
**My main question: Mariner.v3-kata is at the same level as Mariner.v2, Mariner.v3. Does this sound making sense?**
And if the versions array is empty, the for loop in install-dependencies.sh will just ignore it.
Also I will need to use FEATURE_FLAGS to determine if it's a Kata release.

Reply:
This feature is completed